index.js

```diff
@@ -4,16 +4,16 @@ const discovery = require('hyperdiscovery')
 
 module.exports = function (archiveOrKey, opts) {
   var archive = archiveOrKey.key ? archiveOrKey : createArchive(archiveOrKey)
+  var feed = (archive.content) ? archive.content : archive
```

Comment:
It doesn't do anything while [content isn't populated](https://github.com/karissa/hyperhealth/blob/master/index.js#L16). But once content [gets populated after the open](https://github.com/karissa/hyperhealth/blob/master/index.js#L9) (when a connection succeeds and metadata is fetched) then it starts to return the health.